cmd/istioctl/main.go

```diff
@@ -403,6 +408,66 @@ and destination policies.
 		},
 	}
 
+	configCmd = &cobra.Command{
+		Use:   "context-create --api-server http://<ip>:<port>",
+		Short: "Create base kubeconfig file for a server",
```

Comment:
I'd insert something to make it clear it's not to be used for k8s
for instance
```
"Create a kubeconfig for a non kubernetes cluster",
```
or similar (better english smithing welcome but the concept should be there I think)

Reply:
He has left for the day. Will fix in subsequent PR.